options/locale/locale_en-US.ini

```diff
@@ -2012,8 +2013,10 @@ users.auth_login_name = Authentication Sign-In Name
 users.password_helper = Leave the password empty to keep it unchanged.
 users.update_profile_success = The user account has been updated.
 users.edit_account = Edit User Account
-users.max_repo_creation = Maximum Number of Repositories
-users.max_repo_creation_desc = (Enter -1 to use the global default limit.)
+users.max_public_repo_creation = Maximum Number of Public Repositories
+users.max_public_repo_creation_desc = (Enter -1 to use the global default limit.)
+users.max_private_repo_creation = Maximum Number of Private Repositories
+users.max_private_repo_creation_desc = (Enter -1 to use the global default limit.)
```

Comment:
ack

Reply:
I suppose that means "this violates DRY" or some such? :smile: